DQM/L1TMonitor/src/L1TRCT.cc

```diff
@@ -16,22 +16,23 @@
 
 using namespace edm;
 
-const unsigned int PHIBINS = 18;
-const float PHIMIN = -0.5;
-const float PHIMAX = 17.5;
+namespace {
```

Comment:
@Dr15Jones for my understanding, what is the purpose of the definition of this unnamed namespace? Is it to ensure ot restrict the definition of the constants to the scope of this class (through this file as translation unit)?